docs/development/application-registry.md

```diff
@@ -49,6 +49,10 @@ This key lists all models which have been registered in NetBox which are not des
 
 This store maintains all registered items for plugins, such as navigation menus, template extensions, etc.
 
+### `request_processors`
+
+A list of context managers to invoke when processing a request e.g. in middleware or when executing a background job. Request processors can be registered with the `register_request_processor()` decorator.
```

Comment:
Probably don't need the parentheses if the decorator should be applied as `@register_request_processor`.